app/renderer/components/preferences/payment/deletedSites.js

```diff
@@ -26,7 +26,9 @@ class DeletedSitesContent extends ImmutableComponent {
 
   deletePermission (name, hostPattern) {
     appActions.removeSiteSetting(hostPattern, name)
-    this.props.onHide()
+    if (this.props.sites.length === 1) {
```

Comment:
Can you please add check if sites is defined?

Reply:
A check shouldn't be needed, the sites are pulled from [this function](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/preferences/paymentsTab.js#L124) and it will just stay as empty array if no deletedSites are available.

Reply:
From a reusability factor, is it possible another component could call into this without init'ing `sites` first?

Reply:
Normally if we want that one function is used by another component, we would move it into util file. I think we are good here